### ISSUE TITLE
Cannot connect to redis when using sentinel mode and specifying db-id

### ISSUE THREAD

**Issue by ruixuel** [vert-x3/vertx-redis-client#242]
### Version
3.9.2

### Context
I initialize the redis options as:
```
new RedisOptions()
    .setType(RedisClientType.SENTINEL)
    .addConnectionString("redis://127.0.0.1:5000/0")
    .addConnectionString("redis://127.0.0.1:5001/0")
    .addConnectionString("redis://127.0.0.1:5002/0")
    .setMasterName("sentinel7000")
    .setRole(RedisRole.MASTER))
```

However, I got error **No more endpoints in chain**.
And after I debugged the source code, the error is cause by **ERR unknown command `select`, with args beginning with: `0`**
in [https://github.com/vert-x3/vertx-redis-client/blob/master/src/main/java/io/vertx/redis/client/impl/RedisSentinelClient.java#L188](url)
The sentinel redis mode should support select db since it can be used by standalone: 
[https://stackoverflow.com/questions/62037740/does-redis-supports-databases-in-sentinel-setup-with-multiple-nodes](url)

Is there a way to config sentinel redis to support specifying the db-id?

### Steps to reproduce
Create the redis option as below, then try to connect to redis.
```
RedisOptions redisOption = new RedisOptions()
    .setType(RedisClientType.SENTINEL)
    .addConnectionString("redis://127.0.0.1:5000/0")
    .addConnectionString("redis://127.0.0.1:5001/0")
    .addConnectionString("redis://127.0.0.1:5002/0")
    .setMasterName("sentinel7000")
    .setRole(RedisRole.MASTER));
Redis.createClient(vertx, redisOptions)
      .connect(onConnect -> {xxx})
```

**Comment by roostapour** [comment]
@pmlopes Thanks for looking into it. This is blocking us.
`RedisOptions` had a method which was serving the exact purpose. Why is it deprecated now?

**Comment by pmlopes** [comment]
Hi, @roostapour the reasons are briefly described here:
 https://github.com/vert-x3/vertx-4-migration-guide/blob/master/asciidoc/redis.adoc

The current client was hard to maintain and showing its age. No support for cluster, extensions and wire protocol of redis 6.

To solve this we've decided to keep it simple as recommended by redis itself. One of the biggest changes is that redis now operates with connection strings and this change implied that we don't need extra config for password, select etc...

**Comment by mafa73** [comment]
Will this be available in 3.9.x soon?
